### PR TITLE
feature: added card

### PR DESCRIPTION
**Pull request by omgaikwad**
added: card
-Cards with badges
-Cards with dismiss
-Cards with text overlay
-Text only cards
-Vertical/Horizontal cards
-Cards with shadow

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+.card-description {
+  font-size: 12px;
+  margin: 0.4rem 0 0 0.6rem;
+  color: var(--gray);
+}
+.card-footer {
+  margin: 0.4rem 0.4rem 0.4rem 0.6rem;
+  display: flex;
+  align-items: center;
+  /* justify-content: space-around; */
```
Comment (aryan-13) [inline components/card/card.css]: Comments can be removed!

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+.card-close {
+  position: absolute;
+  top: 8px;
+  right: 12px;
+  cursor: pointer;
+  color: var(--gray);
+}
+
+.card-overlay {
+  /* display: none; */
```
Comment (aryan-13) [inline components/card/card.css]: Here also

**Review by aryan-13 — COMMENTED** [review]
Else everything looks fine

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  margin: 1rem;
+  background-color: var(--white);
+  color: var(--secondary-color);
+  box-shadow: 4px 4px lightgray;
+}
+
+.card-horizontal {
+  display: flex;
+  position: relative;
+  flex-direction: row;
```
Comment (Tanay-J) [inline components/card/card.css]: ```row``` is default ```flex-direction```. You may remove this line. 

Everything else LGTM

**Review by vijaytembugade — APPROVED** [review]
Looks cool. I hope you will work on responsivemess later.

File: components/card/card.html [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+        </div>
```
Comment (amarphule) [inline components/card/card.html:179]: all frames code are not visible properly,

File: components/card/card.css [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  width: 100%;
+  height: 70%;
+}
+.card-img img {
+  width: 100%;
+  height: 100%;
+}
+
+#card-text-only-desc {
+  font-size: 14px;
```
Comment (hinanshisuthar) [inline components/card/card.css]: Use rems if the value is >=10 px

File: components/card/card.css [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+  color: var(--secondary-color);
+}
+
+.card-badge {
+  position: absolute;
+  background-color: var(--primary-color);
+  color: var(--white);
+  width: fit-content;
+  height: fit-content;
+  font-size: 14px;
```
Comment (hinanshisuthar) [inline components/card/card.css]: here too. Also, define a variable of font size: 14 for the redundancy

**Review by hinanshisuthar — CHANGES_REQUESTED** [review]
Everything else looks fine.

File: components/card/card.css (hunk 1 above)
Comment (omgaikwad) [reply components/card/card.css]: I will remove it.

File: components/card/card.css (hunk 2 above)
Comment (omgaikwad) [reply components/card/card.css]: Yes

File: components/card/card.css (hunk 3 above)
Comment (omgaikwad) [reply components/card/card.css]: Yes sure I will remove it

File: components/card/card.html (hunk 4 above)
Comment (omgaikwad) [reply components/card/card.html:179]: Frames code are "scrollable",  can you just check once again and suggest if there still problem

File: components/card/card.css (hunk 5 above)
Comment (omgaikwad) [reply components/card/card.css]: Sure, I will change it to rems

File: components/card/card.css (hunk 6 above)
Comment (omgaikwad) [reply components/card/card.css]: Yeah i will change it

**Pull request merged**